### PR TITLE
Disabling multi-selection in explorer

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

disabling multi selection in farms explorer

### Changes

removing filter button, filter by last input only.

### Related Issues

- https://github.com/threefoldtech/tfgrid_dashboard/issues/170

### CODE REVIEW

File: src/explorer/views/Farms.vue [hunk 1]
```diff
@@ -219,26 +196,26 @@ export default class Farms extends Vue {
...
-        case "id":
-          orderBy.push(desc ? "farmID_DESC" : "farmID_ASC");
-          break;
-        case "name":
-          orderBy.push(desc ? "name_DESC" : "name_ASC");
-          break;
-      }
-    }
-    _vars.orderBy = orderBy.length === 0 ? undefined : orderBy;
+    //   const orderBy: string[] = [];
```
Comment (sameh-farouk) [inline src/explorer/views/Farms.vue]: can we clean old code instead of commenting it

Comment (zaelgohary) [reply src/explorer/views/Farms.vue]: done.

File: src/explorer/views/Farms.vue [hunk 2]
```diff
@@ -121,7 +98,7 @@ import getFarmPublicIPs from "../utils/getFarmPublicIps";
 import gql from "graphql-tag";
 import equalArrays from "../utils/equalArrays";
 import LayoutFilters from "../components/LayoutFilters.vue";
-import { execute } from 'graphql';
+import { execute } from "graphql";
```
Comment (sameh-farouk) [inline src/explorer/views/Farms.vue]: no longer used

File: src/explorer/views/Farms.vue [hunk 3]
```diff
@@ -104,13 +87,7 @@
...
 import { Component, Vue, Watch } from "vue-property-decorator";
 import DetailsV2 from "../components/DetailsV2.vue";
-import {
-  filterQuery,
-  getFarmsQuery,
-  IFarm,
-  IFetchPaginatedData,
-  IFilterQuery,
-} from "../graphql/api";
+import { filterQuery, getFarmsQuery, IFarm, IFetchPaginatedData, IFilterQuery } from "../graphql/api";
```
Comment (sameh-farouk) [inline src/explorer/views/Farms.vue]: `filterQuery`, `IFilterQuery` and `apollo` are no longer used and should be removed

**Review by sameh-farouk — COMMENTED** [review]
we don't need these imports
 'filterQuery', 'IFilterQuery', 'apollo', and  'execute'

File: src/explorer/views/Farms.vue (hunk 2 above)
Comment (zaelgohary) [reply src/explorer/views/Farms.vue]: done.

File: src/explorer/views/Farms.vue (hunk 3 above)
Comment (zaelgohary) [reply src/explorer/views/Farms.vue]: done.

**Review by sameh-farouk — APPROVED** [review]

**Pull request merged**